File: PY/even_or_odd_counter.py
#Write a program that asks the user to input 10 integers. 
# Your program should count and display:

##How many of the integers are even
##How many of the integers are odd

import os

def clear_screen():
    os.system("clear")

def odd_even_count():
    print("Welcome to the Even or Odd Counter!")
    even_count = 0
    odd_count = 0

    numbers = []

    for i in range(10):
        while True:
            try:
                num = int(input(f"Enter number {i + 1}: "))
                break
            except ValueError:
                print("Enter a valid number!")
        
        numbers.append(num)

        if num % 2 == 0:
            even_count += 1
        else:
            odd_count += 1

    print(f"\nNumbers entered {numbers}")

    print(f"You entered {odd_count} odd numbers and {even_count} even numbers")

while True:

    clear_screen()

    odd_even_count()

    choice = input("Do you wish to play again? y/n: ")
    if choice != "y":
        break

File: C#/EvenOrOddCounter.cs
//Write a program that asks the user to input 10 integers. 
//Your program should count and display:

//How many of the integers are even
//How many of the integers are odd

using System;
using System.Runtime.InteropServices;

public class EvenOrOddCounter
{
    public static void Run()
    {
        while (true)
        {
            Console.Clear();

            Console.WriteLine("Welcome to the Even or odd counter");

            int evenCount = 0;
            int oddCount = 0;
            List<int> numbers = new List<int>();

            for (int i = 0; i < 10; i++)
            {
                Console.Write($"Enter number {i + 1}: ");
                int number = int.Parse(Console.ReadLine()!);
                numbers.Add(number);

                if (number % 2 == 0)
                {
                    evenCount += 1;
                }
                else
                    oddCount += 1;
            }

            Console.WriteLine($"You entered {oddCount} odd numbers and {evenCount} even numbers");
            Console.Write("Do you wish to play again? y/n :");

            string choice;
            choice = Console.ReadLine()!;

            if (choice != "y")
            {
                break;
            }
        }        
    }
}



